docs/src/main/sphinx/release/release-365.md

```diff
@@ -0,0 +1,144 @@
+# Release 365 (3 Dec 2021)
+
+## General
+
+* Add support for [`TRUNCATE TABLE`](/sql/truncate). ({issue}`8921`)
+* Add support for aggregate functions in row pattern recognition context. ({issue}`8738`)
+* Add support for time travel queries. ({issue}`8773`)
+* Add support for spilling aggregations containing `ORDER BY` or `DISTINCT` clauses. ({issue}`9723`)
```

Comment:
Does it have any session or config toggles?

Reply:
Yes, but it's enabled by default.